src/main/java/ro/secur/auth/configuration/CorsConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package ro.secur.auth.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class CorsConfiguration {
+
+    @Bean
```

Comment:
asta e adaugata de mine in SecurityConfiguration pe dev. nu cred ca mai ai nevoie de ea. o poti sterge.